WebRoot/skins/_base/base4/skin.properties

```diff
@@ -921,8 +921,8 @@ ListItemBgColor-hover           = @lighten(AppC,25)@
 ListItemBgColor-active          = @lighten(AppC,25)@
 ListItemBgColor-matched         = @MatchedColor@
 ListItemBgColor-selected        = @AltC@
-ListItemBgColor-sel-alt         = @SelAltColor@
-ListItemBgColor-actioned        = @SelC@
+ListItemBgColor-sel-alt         = @AltC@
+ListItemBgColor-actioned        = @hexToRGBA(SecondaryBrandingColor, 10)@
```

Comment:
Can we have solid color instead of `rgba`. We had once changed `rgba` to solid in past if you remember.

Thanks

Reply:
Fixed in https://github.com/Zimbra/zm-web-client/pull/163/commits/4839250d72ed00dd9a882eb6499c282f7f5a896d